src/pages/settings/Profile/ProfilePage.js

```diff
@@ -226,19 +218,25 @@ class ProfilePage extends Component {
                             <View style={styles.mb1}>
                                 <ExpensiPicker
                                     label={this.props.translate('profilePage.preferredPronouns')}
-                                    onChange={pronouns => this.setState({pronouns, selfSelectedPronouns: ''})}
-                                    items={this.pronounDropdownValues}
+                                    onChange={(pronouns) => {
+                                        const selfSelectedPronouns = pronouns === CONST.PRONOUNS.SELF_SELECT;
```

Comment:
Similar here, I think it makes sense to make the variable name more clearly a boolean, like `areSelfSelectedPronouns`

Reply:
Done! I used `hasSelfSelectedPronouns` in both places.